src/components/screens/signMultiSignTransaction/importData/importData.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable complexity */
+import React, { useState, useEffect } from 'react';
+import Box from '../../../toolbox/box';
+import BoxContent from '../../../toolbox/box/content';
+import BoxFooter from '../../../toolbox/box/footer';
+import { PrimaryButton } from '../../../toolbox/buttons';
+import Feedback from '../../../toolbox/feedback/feedback';
+import ProgressBar from '../progressBar';
+import styles from './styles.css';
+
+const reader = new FileReader();
+
+// eslint-disable-next-line max-statements
+const inputValidator = ({
+  senderId, senderPublicKey, fee, asset,
+  nonce, signatures, lsTrackingId, id,
+}) => {
+  const validation = {
+    valid: true,
```

Comment:
Do we really need a boolean here? Can we not just directly check `errors.length`?

Reply:
In the future we will have to replace this validation for `Lisk.utils.validateMultisignatures` which returns an object like this. This is why I decided to keep that structure.